haskell/test/SMT/RegistryDAO.hs

```diff
@@ -47,8 +47,7 @@ hprop_RegistryDaoSMT =
       , soRejectedProposalSlashValue = registryDaoRejectedProposalSlashValue
       , soDecisionLambda = registryDaoDecisionLambda
       , soCustomEps = Map.fromList
-          [ ( unsafeMkMText "receive_xtz", \_ -> pure () )
```

Comment:
I see that you have removed `receive_xtz` from the custom entrypoints in the tests, but not in the ligo code. Was this intentional?

I would imagine that with `default` we no longer need it at all.

Related: both in `docs/registry.md` and `docs/treasury.md` there is an explanation of the `receive_xtz` entrypoint left behind.

Reply:
Ah, good catch. I rmember thinking that custom LIGO entrypoints where dynamic (encoding as bytes) and thus does not require removing anything.

Reply:
I think we also need to mention the `default` entrypoint in the main spec.

Reply:
Have removed LIGO entrypoints, updated docs and mentioned `default` entrypoint in main spec.